src/pages/index.js

```diff
@@ -41,7 +41,7 @@ const features2 = [
   {
     title: "Self-Managed",
     imageUrl: "img/self-managed.png",
-    url: "/docs/self-managed/about-self-managed/",
+    url: "/",
```

Comment:
This URL has nothing related in 0.25! Instead of removing the entire feature block, because that makes it look funny, I left it in and pointed it at the home page. It's "unsupported" so that's safe????